applications/ContactStructuralMechanicsApplication/python_scripts/contact_structural_mechanics_analysis.py

```diff
@@ -1,80 +0,0 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
-
-# Importing Kratos
-import KratosMultiphysics as KM
-
-# Other imports
-import sys
-
-# Import the base structural analysis
-from KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_analysis import StructuralMechanicsAnalysis as BaseClass
-from KratosMultiphysics.StructuralMechanicsApplication import python_solvers_wrapper_structural
-
-class ContactStructuralMechanicsAnalysis(BaseClass):
```

Comment:
BTW I think you could easily achieve backward-compatibility
If you remove it completely it will break all your cases